docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-docker-installation.md

```diff
@@ -0,0 +1,184 @@
+# Admin API 1.x - Docker installation
+
+**Contents:**
+
+* [Before You Install](#before-you-install)
+  * [Compatibility & Supported ODS / API Versions](#compatibility-supported-ods-api-versions)
+* [Installation Instructions](#installation-instructions)
+  * [General Prerequisites](#general-prerequisites)
+* [Installation Instructions](#installation-instructions)
+  * [1\. Include Admin API in the ODS Docker Setup](#1-include-admin-api-in-the-ods-docker-setup)
+  * [2\. Relaunch the Docker Composition](#2-relaunch-the-docker-composition)
+  * [3\. Execute First-Time Configuration](#3-execute-first-time-configuration)
```

Comment:
In the ODS/API documentation, I have been removing these introductory TOC's, since Docusaurus builds a TOC sidebar on the right automatically. That's up to you though.